source/Cosmos.HAL2/Global.cs

```diff
@@ -46,15 +46,24 @@ static public void Init(TextScreenBase textScreen)
 
       mDebugger.Send("Done initializing Cosmos.HAL.Global");
 
-      mDebugger.Send("ATA Primary Master");
-      InitAta(Ata.ControllerIdEnum.Primary, Ata.BusPositionEnum.Master);
-
-      //TODO Need to change code to detect if ATA controllers are present or not. How to do this? via PCI enum?
-      // They do show up in PCI space as well as the fixed space.
-      // Or is it always here, and was our compiler stack corruption issue?
-      mDebugger.Send("ATA Secondary Master");
-      InitAta(Ata.ControllerIdEnum.Secondary, Ata.BusPositionEnum.Master);
-      //InitAta(BlockDevice.Ata.ControllerIdEnum.Secondary, BlockDevice.Ata.BusPositionEnum.Slave);
+      if (PCI.GetDeviceClass(0x01, 0x01) != null)
```

Comment:
Please add a comment about the meaning of these numbers and the ones on line 57 too.